cmd/control/console_init.go

```diff
@@ -70,6 +70,15 @@ func consoleInitFunc() error {
 	createHomeDir(rancherHome, 1100, 1100)
 	createHomeDir(dockerHome, 1101, 1101)
 
+	// who & w command need this file
+	if _, err := os.Stat("/run/utmp"); os.IsNotExist(err) {
+		f, err := os.OpenFile("/run/utmp", os.O_RDWR|os.O_CREATE, 0644)
```

Comment:
Why not `os.Create`? Any special reason?

Reply:
os.Create perm is 0666. I need 0644.